src/QrPayment.php

```diff
@@ -432,14 +433,28 @@ public function setXzBinaryLocator(XzBinaryLocatorInterface $xzBinaryLocator): Q
         return $this;
     }
 
+    /**
+     * @param string|null $binaryPath
+     *
+     * @return WindowsXzBinaryLocator|LinuxXzBinaryLocator
+     */
+    public function getXzBinaryLocatorByOs(?string $binaryPath = null)
```

Comment:
- Please add return type of `XzBinaryLocatorInterface`, both in phpdoc and in the method signature.
- Change method visibility to private